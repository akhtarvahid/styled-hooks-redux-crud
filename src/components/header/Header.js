import React from "react";
import './header.scss';
import logo from '../../assets/logo.gif';
function Header({ addHandle }) {
  return (
    <div className="menu-header">
      <div className="menu-header__menu-inner">
        <div>
          <img
            alt="logo"
            src={logo}
            width="80"
            height="50"
            className="d-inline-block align-top"/>
          <div>REACT CRUD</div>
        </div>
        <button onClick={() => addHandle()}>Add</button>
      </div>
    </div>
  );
}

export default Header;
