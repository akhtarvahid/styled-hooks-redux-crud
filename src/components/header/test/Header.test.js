import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

test('should render Header Component', () => {
    const tree = renderer
        .create(<Header />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
