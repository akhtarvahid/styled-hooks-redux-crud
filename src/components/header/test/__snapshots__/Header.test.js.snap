// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header Component 1`] = `
<div
  className="menu-header"
>
  <div
    className="menu-header__menu-inner"
  >
    <div>
      <img
        alt="logo"
        className="d-inline-block align-top"
        height="50"
        src="logo.gif"
        width="80"
      />
      <div>
        REACT CRUD
      </div>
    </div>
    <button
      onClick={[Function]}
    >
      Add
    </button>
  </div>
</div>
`;
